.github/labeler.yml

```diff
@@ -18,10 +18,10 @@
 - changed-files:
   - any-glob-to-any-file: proof_gen/**
 
-# Add 'crate: zero_bin' label to any changes within 'zero_bin' folder.
+# Add 'crate: zero_bin' label to any changes within 'zero' folder.
 'crate: zero_bin':
```

Comment:
nit: we may want to rename the lint to `crate: zero` as well?

Reply:
I kept this the same for convenience (e.g looking at historical issues). Happy to change :)